pkg/transparentproxy/validate/validator.go

```diff
@@ -161,7 +161,8 @@ func (v *Validator) RunClient(
 			return nil
 		},
 	); err != nil {
-		return errors.Wrap(err, "client failed to connect to the verification server after retries")
+		return errors.Wrap(err, "client failed to connect to the verification server after retries - " +
+			"most likely iptables rules are not applied correctly, please check CNI logs")
```

Comment:
```suggestion
			"most likely the transparent proxy is not established correctly, please check CNI logs")
```

Reply:
Well a more general wording better? So that it still applies to other approaches of setting up the TP, like ebpf